modules/oadp-usecase-include-ca-cert-backup.adoc

```diff
@@ -0,0 +1,276 @@
++
+[source,terminal]
+----
+$ oc extract --to=- cm/test-obc # <1>
+----
+<1> `test-obc` is the name of the OBC.
++
+.Example output
+[source,terminal]
+----
+# BUCKET_NAME
+backup-c20...41fd
+# BUCKET_PORT
+443
+# BUCKET_REGION
+
+# BUCKET_SUBREGION
+
+# BUCKET_HOST
+s3.openshift-storage.svc
+----
+
+. To get the bucket credentials from the generated `secret`, run the following command:
++
+[source,terminal]
+----
+$ oc extract --to=- secret/test-obc 
+----
++
+.Example output
+[source,terminal]
+----
+# AWS_ACCESS_KEY_ID
+ebYR....xLNMc
+# AWS_SECRET_ACCESS_KEY
+YXf...+NaCkdyC3QPym
+----
+
+. Create a `cloud-credentials` file with the object bucket credentials as shown in the following command:
++
+[source,terminal]
+----
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID> 
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY> 
+----
+
+. Create the `cloud-credentials` secret with the `cloud-credentials` file content as shown in the following command:
++
+[source,terminal]
+----
+$ oc create secret generic \
+  cloud-credentials \
+  -n openshift-adp \
+  --from-file cloud=cloud-credentials
+----
+
+. Extract the service CA certificate from the `openshift-service-ca.crt` config map by running the following command. Ensure that you encode the certificate in `Base64` format and `echo` the value to use in a later step.
++
+[source,terminal]
+----
+$ oc get cm/openshift-service-ca.crt \
+  -o jsonpath='{.data.service-ca\.crt}' | base64 -w0; echo
+----
++
+.Example output
++
+[source,terminal]
+----
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0...
+....gpwOHMwaG9CRmk5a3....FLS0tLS0K
+----
+
+. Configure the Data Protection Application (DPA) with the bucket name and CA certificate as shown in the following example:
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true 
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc 
+          s3ForcePathStyle: "true" 
+          insecureSkipTLSVerify: "false" # <1>
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials 
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: oadp
+          caCert: <ca_cert> # <3>
+----
+<1> The `insecureSkipTLSVerify` flag can be set to either `true` or `false`. If set to "true", SSL/TLS security is disabled. If set to `false`, SSL/TLS security is enabled.
+<2> Specify the name of the bucket extracted in an earlier step.
+<3> Copy and paste the `Base64` encoded certificate from an earlier step.
+
+. Create the DPA by running the following command:
```

Comment:
Since I am not following the suggestion to create a yaml file, 'Create the DPA' makes more sense and is consistent with other doc sets. Thanks for suggestion.